firestore/google/cloud/firestore_v1beta1/document.py

```diff
@@ -420,17 +419,11 @@ def get(self, field_paths=None, transaction=None):
         Returns:
             ~.firestore_v1beta1.document.DocumentSnapshot: A snapshot of
             the current document.
```

Comment:
Needs a note about the state of  a missing document (empty `data`, `create_time` and `update_time` are `None`).